corehq/form_processor/reprocess.py

```diff
@@ -48,7 +48,14 @@ def reprocess_unfinished_stub(stub, save=True):
         return
 
     assert form.is_normal
-    result = _reprocess_form(form, save)
+    try:
+        result = _reprocess_form(form, save)
+    except:
+        if save:
+            stub.queued = False
+            stub.save()
```

Comment:
it's basically representing how often to requeue tasks that fail hard. it will always queue every item the first time it sees it, and if any of them fail hard then it won't requeue them until this amount of time has passed. so we could even set it to a week, meaning in the small chance that something fails hard we just don't retry it until a week later.

i don't think we can reliably use a `queued` flag because we can't guarantee that we can unset it in every failure scenario. a queued timestamp could work, where we always include items that haven't been queued in a long time, but that's the same thing as setting a longer `get_enqueuing_timeout` value.

to use a longer `get_enqueuing_timeout` value we would also have to remove the [:1000] limit on `get_items_to_be_processed`, but we could use a larger value for `get_fetching_interval` so we don't hit the db so often.